userapi/src/main/java/fk/prof/userapi/StorageFactory.java

```diff
@@ -25,27 +26,28 @@
 public class StorageFactory {
 
   private static final String S3 = "s3";
-    private static final String ACCESS_KEY = "access.key";
-    private static final String SECRET_KEY = "secret.key";
-    private static final String END_POINT = "endpoint";
+  private static final String ACCESS_KEY = "access.key";
+  private static final String SECRET_KEY = "secret.key";
+  private static final String END_POINT = "endpoint";
+  private static final String LIST_OBJECTS_TIMEOUT_MS = "list.objects.timeout.ms";
 
-    private static S3AsyncStorage s3AsyncStorage = null;
+  private static S3AsyncStorage s3AsyncStorage = null;
 
   synchronized public static AsyncStorage getAsyncStorage(JsonObject storageConfig, ExecutorService storageExecSvc) {
-    if (storageConfig.fieldNames().isEmpty())
-      return getS3AsyncInstance(storageConfig.getJsonObject(S3), storageExecSvc);
-    switch (storageConfig.fieldNames().iterator().next()) {
-            case S3:
-              return getS3AsyncInstance(storageConfig.getJsonObject(S3), storageExecSvc);
-            default:
-              return getS3AsyncInstance(storageConfig.getJsonObject(S3), storageExecSvc);
-        }
+    for (String key : storageConfig.fieldNames()) {
```

Comment:
Removed the class. After refactoring, the class has been moved to storage module.